src/prefect/_internal/concurrency/executor.py

```diff
@@ -36,30 +36,30 @@ class Executor(BaseExecutor):
     def __init__(
         self,
         worker_type: WorkerType = "thread",
-        **worker_pool_kwargs: Any,
+        **work_pool_kwargs: Any,
```

Comment:
I think the worker pools in this file are a different type of worker pool and probably shouldn't be included in this name change.